Mathlib/Analysis/Convex/Between.lean

```diff
@@ -810,6 +810,18 @@ theorem wbtw_iff_sameRay_vsub {x y z : P} : Wbtw R x y z ↔ SameRay R (y -ᵥ x
     field_simp [(add_pos hr₁ hr₂).ne', hr₂.ne']
     ring
 
+theorem AffineIndependent.not_wbtw {a b c : P} (hT : AffineIndependent R ![a, b, c]) :
+    ¬ Wbtw R a b c := by
+  rw [affineIndependent_iff_not_collinear_set] at hT
+  exact mt (·.collinear) hT
```

Comment:
Note that the `Wbtw.collinear` lemma is defined just ~50 lines above here, in the same section. That fact makes me feel like this is a good home for these new two lemmas.